features/call/src/main/kotlin/io/element/android/features/call/ui/CallScreenPresenter.kt

```diff
@@ -90,10 +94,19 @@ class CallScreenPresenter @AssistedInject constructor(
                     .onEach {
                         // Relay message to Widget Driver
                         callWidgetDriver.value?.send(it)
+                        println(it)
```

Comment:
Strange to see `println` here. Use Timber or remove the temporary log?

Reply:
Nice catch, I deleted a couple of other temporary logs, but I forgot about this one.